### PR TITLE
Minimal PeertubeModalService to open settings from "can be redefined..."

### LINKED ISSUE #2576: Make the "can be redefined by the users" text to be a hyperlink and link to the settings option where the user can redefine that
Make the "can be redefined by the users" text (from the About page) to be a hyperlink and link to the settings option where the user can redefine that. We can make it just open the settings page or we can make it open the settings page and flash two times (with a somewhat bright color) the option in question. 

This issue concerns both the registered and unregistered users.

### PR DESCRIPTION
**Pull request by aschaap**
## Description

Fixes #2576.

## Has this been tested?

- [ ] 👍 yes, I added tests to the test suite
- [ ] 👍 yes, light tests as follows are enough
- [ ] 💭 no, because this PR is a draft and still needs work
- [X] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

In the dev environment, open the About page and click on "can be redefined by the users". The expected result is that the settings dialog opens.

### CODE REVIEW

File: client/src/app/menu/menu.component.ts [hunk 1]
```diff
@@ -130,6 +132,9 @@ export class MenuComponent implements OnInit {
         this.authService.userInformationLoaded
           .subscribe(() => this.buildUserLanguages())
       })
+
+    this.modalService.openQuickSettingsSubject
+      .subscribe((msg) => this.openQuickSettings())
```
Comment (Chocobozzz) [inline client/src/app/menu/menu.component.ts:137]: `() => this.openQuickSettings`

File: client/src/app/shared/shared-instance/instance-features-table.component.ts [hunk 2]
```diff
@@ -11,7 +12,10 @@ export class InstanceFeaturesTableComponent implements OnInit {
   quotaHelpIndication = ''
   serverConfig: ServerConfig
 
-  constructor (private serverService: ServerService) { }
+  constructor (
+    private serverService: ServerService,
+    private modalService: PeertubeModalService
+    ) { }
```
Comment (Chocobozzz) [inline client/src/app/shared/shared-instance/instance-features-table.component.ts:18]: Align `) { }` with `constructor`

File: client/src/app/shared/shared-main/peertube-modal/peertube-modal.service.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Injectable } from '@angular/core'
+import { Subject } from 'rxjs'
+
+@Injectable({ providedIn: 'root' })
+export class PeertubeModalService {
+  openQuickSettingsSubject = new Subject<any[]>()
```
Comment (Chocobozzz) [inline client/src/app/shared/shared-main/peertube-modal/peertube-modal.service.ts:6]: Should be `new Subject<void>()`

**Review by Chocobozzz — CHANGES_REQUESTED** [review]

File: client/src/app/menu/menu.component.ts (hunk 1 above)
Comment (aschaap) [reply client/src/app/menu/menu.component.ts]: Thank you for your prompt review!

**Comment by aschaap** [comment]
Failing api-2 test:
1) Should have a valid playlist oEmbed response

Failing api-3 tests:
1) Should have the correct thumbnails
1) Should automatically update the thumbnail when adding an element
1) Should create a playlist on server 2 and have the playlist on server 1 but not on server 3

@Chocobozzz, I'm not confident these failing tests are related to the changes you suggested; perhaps there is an issue somewhere in the testing infrastructure?

**Comment by Chocobozzz** [comment]
Thanks for the PR and sorry for the tests, my bad.

**Pull request merged**